pkgs/development/python-modules/warp-lang/default.nix

```
@@ -53,24 +53,27 @@ let
     src =
       let
         baseURL = "https://developer.download.nvidia.com/compute/cublasdx/redist/cublasdx";
+        cudaVersion = "12.0";
+        cudaMajorVersion = lib.head (lib.splitString "." cudaVersion);
```

```suggestion
        cudaMajorVersion = cudaPackages.cudaMajorVersion;
        cudaVersion = "${cudaMajorVersion}.0";
```

Maybe add a note that the URL has the version end in `.0` for some reason.